conductor/lib/common.py

```diff
@@ -574,14 +573,32 @@ def load_resources_file():
 
 def get_conductor_instance_types(as_dict=False):
     '''
-    Get the list of available instances types from the resources.yml file
+    Get the list of available instances types.
     '''
-    resources = load_resources_file()
-    instance_types = resources.get("instance_types") or []
+    from conductor.lib import api_client
+    api = api_client.ApiClient()
 
-    if as_dict:
-        return dict([(instance["name"], instance) for instance in instance_types])
+    from conductor.lib.downloader import get_bearer_token
```

Comment:
And if we're going to reuse a function from another application (i.e. the downloader), can you please extract the function from that application and put it in a library module (`api_client`) for all apps to use.